src/Components/SignUpForm.js

```diff
@@ -172,7 +173,7 @@ const SignUpForm = ({
           </FlexContainer>
         </form>
         <Paragraph appearance="white">
-          Already have an account? <a href="#">Log in here!</a>
+          Already have an account? <Link to="/login">Log in here!</Link>
```

Comment:
This commit references an implementation of a Redirect, but I can't see anything. Something missing?